bundles/org.openhab.binding.freecurrency/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="freecurrency"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="info" extensible="exchange-rate">
```

Comment:
`info` is a bit too generic and may not age well if the API starts providing more info. So perhaps it can be more specific?

I see the API differentiates between latest and historical exchange rates so maybe that can be reflected in the name?

Reply:
In general I agree. But unfortunately I'm out of ideas what would be a better naming.